src/cim/components/Hypershift/HostedClusterWizard/HostsStep/HostsAdvancedSelection.tsx

```diff
@@ -66,7 +67,7 @@ const HostsAdvancedSelection: React.FC<HostsAdvancedSelectionProps> = ({
   );
 
   const paginationProps = usePagination(hosts.length);
-
+  const { t } = useTranslation();
```

Comment:
The call to `hostNameColumn` is missing the `t` prop.
I'd move the  `useTranslation` a few lines up so you can pass it to that function. 

```
  const { t } = useTranslation();

  const content = React.useMemo(
    () => [hostnameColumn(t, undefined, hosts), cpuCoresColumn, memoryColumn, disksColumn],
    [hosts],
  );
```

Reply:
Ok!